src/Tests/Backend/Backend.test.ts

```diff
@@ -53,22 +56,39 @@ class ExecutorMockup extends ExecutorBase {
 
 suite('Backend', function() {
   suite('backendRegistrationApi', function() {
+    test('registers a ONEToolchain', function() {
+      let oneBackend = new ONEToolchain();
```

Comment:
```suggestion
      // Register ONE Toolchain backend
      backendRegistrationApi();
      let oneBackend = new ONEToolchain();
```

Oh, I'm sorry. When we remove the `backendRegistrationApi()` function, the PR test is failed because ONE backend was not registered.